docs/asides.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Asides
+description: Asides provide an easy way to represent content that is only indirectly related to the article's main content. 
+---
+
+## Asides
+
+Asides provide an easy way to represent content that is only indirectly related to the article's main content. Where supported, MyST will attempt to display an `aside` _as close as possible_ but separately from the main article, such as in the side-margin. 
+
+````{myst}
+The main article contains main-article text!
+
+```{aside}
+This is an aside. It is not entirely relevant to the main article.
+```
+````
```

Comment:
This one might be easier to start from the jupyter-books docs on this and demonstrate it in the page rather than in a demo component?

Reply:
I think an `aside` should be a MyST feature, but I would not be surprised if `aside` isn't very clear in the demo component. We can enable a grid layout on the MyST demo component if needs be.

Welcome your thoughts here if you think it would be better not to use the `myst` directive and instead include the `aside` in the main guide body. But, I think we can fix-up the demo component to make it possible to see, probably.